qchem/pennylane_qchem/qchem.py

```diff
@@ -690,6 +690,94 @@ def hf_state(n_electrons, m_spin_orbitals):
     return np.array(hf_state_on)
 
 
+def ph_pphh_wires(ph_confs, pphh_confs, wires=None):
+
+    r"""Map the indices representing the particle-hole configurations
+    generated by the Coupled-Cluster excitation operator to the wires that
+    the Unitary Coupled-Cluster Singles and Doubles (UCCSD) template will act on.
```

Comment:
```suggestion
    can be fed into the Unitary Coupled-Cluster Singles and Doubles (UCCSD) template.
```

Not sure if this is better?